test/functional/terminal/scrollback_spec.lua

```diff
@@ -441,8 +439,7 @@ describe("'scrollback' option", function()
     -- 'scrollback' option is synchronized with the internal sb_buffer.
     command('sleep 100m')
     if iswin() then
-      nvim('command',
-      'call jobsend(b:terminal_job_id, "for($i=1;$i -le 40;$i++){Write-Host \\"line$i\\"}\\<CR>")')
+      feed_data('for($i=1;$i -le 40;$i++){Write-Host \"line$i\"}\13') -- \13 = <CR>
```

Comment:
If I used ``` \n```, the command will not run. Therefore, I was using ```nvim('command', 'call jobsedn(...)")```.  If I used ```\13```, the command will run, from that I found out that I can use ```\r```. So I changed it to use ```\r```.